### PR TITLE
Adding discovery 8.0 docs

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

File: plugins/foreman_discovery/8.0/index.md [hunk 1]
```diff
@@ -0,0 +1,1302 @@
...
+
+## 3.1.1 Default PXE template
+
+In the Foreman UI, go to Provisioning Templates, edit *PXELinux global
+default* template and change the default entry from localboot entry to
+Discovery:
+
+    ONTIMEOUT discovery
+
+The template already contains a menu entry similar to this and nothing needs
```
Comment (dLobatog) [inline plugins/foreman_discovery/8.0/index.md]: The template actually just contains `<%= snippet "pxelinux_discovery" %>`, best to tell people to take a look there.

File: plugins/foreman_discovery/8.0/index.md [hunk 2]
```diff
@@ -0,0 +1,1302 @@
...
+title: Foreman Discovery 8.0 Manual
+version: 8.0
+# versions for matrix and snippets
+# (use short version for imgver e.g. 3.0)
+pluginver: 8.0.0
+proxyver: 1.0.3
+imgver: 3.3
+cliver: 0.0.3
+# uncomment to show warning box for an old release
+#warning: old
```
Comment (domcleal) [inline plugins/foreman_discovery/8.0/index.md:13]: Should this be set in 7.0?

File: plugins/foreman_discovery/8.0/index.md [hunk 3]
```diff
@@ -0,0 +1,1302 @@
...
+  </tr>
+  <tr>
+    <td>= 1.13</td>
+    <td>7.0.0</td>
+    <td>1.0.3</td>
+    <td>3.2-3.3</td>
+    <td>0.0.3</td>
+  </tr>
+  <tr>
+    <td>= 1.13</td>
```
Comment (domcleal) [inline plugins/foreman_discovery/8.0/index.md]: 1.14?

File: plugins/foreman_discovery/8.0/index.md [hunk 4]
```diff
@@ -0,0 +1,1302 @@
+---
+layout: plugin
+pluginname: foreman_discovery
+title: Foreman Discovery 8.0 Manual
+version: 8.0
+# versions for matrix and snippets
+# (use short version for imgver e.g. 3.0)
+pluginver: 8.0.0
+proxyver: 1.0.3
```
Comment (domcleal) [inline plugins/foreman_discovery/8.0/index.md]: 1.0.4?

**Review by domcleal — CHANGES_REQUESTED** [review]

File: plugins/foreman_discovery/8.0/index.md (hunk 1 above)
Comment (lzap) [reply plugins/foreman_discovery/8.0/index.md]: Yeah, but I prefer to be quite verbose here, there's been so many issues in the past with configuring this one. I will change the wording tho to refer to the snippet tho.

File: plugins/foreman_discovery/8.0/index.md (hunk 2 above)
Comment (lzap) [reply plugins/foreman_discovery/8.0/index.md:13]: Yeah I was thinking 1.14 is not yet out but this is irrelevant. Will change this and the others thanks.

**Comment by lzap** [comment]
The changes are up!

**Review by domcleal — APPROVED** [review]

**Comment by lzap** [comment]
@dLobatog any other comments?

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**